cime_config/config_grids.xml

```diff
@@ -4823,6 +4859,31 @@
       <map name="GLC2OCN_SMAPNAME">cpl/gridmaps/mpas.gis20km/map_mpas.gis20km_to_oEC60to30v3_aave.181115.nc</map>
     </gridmap>
 
+    <gridmap glc_grid="mpas.gis20km" lnd_grid="ne30np4.pg2">
+      <map name="LND2GLC_FMAPNAME">cpl/gridmaps/ne30pg2/map_ne30pg2_to_gis20km_mono.230510.nc</map>
+      <map name="LND2GLC_SMAPNAME">cpl/gridmaps/ne30pg2/map_ne30pg2_to_gis20km_bilin.230510.nc</map>
+      <map name="GLC2LND_FMAPNAME">cpl/gridmaps/mpas.gis20km/map_gis20km_to_ne30pg2_mono.230510.nc</map>
+      <map name="GLC2LND_SMAPNAME">cpl/gridmaps/mpas.gis20km/map_gis20km_to_ne30pg2_mono.230510.nc</map>
+    </gridmap>
+
+    <gridmap glc_grid="mpas.gis20km" lnd_grid="r05">
+      <map name="LND2GLC_FMAPNAME">cpl/gridmaps/r05/map_r05_to_gis20km_mono.230510.nc</map>
+      <map name="LND2GLC_SMAPNAME">cpl/gridmaps/r05/map_r05_to_gis20km_bilin.230510.nc</map>
+      <map name="GLC2LND_FMAPNAME">cpl/gridmaps/mpas.gis20km/map_gis20km_to_r05_mono.230510.nc</map>
+      <map name="GLC2LND_SMAPNAME">cpl/gridmaps/mpas.gis20km/map_gis20km_to_r05_mono.230510.nc</map>
+    </gridmap>
+
+    <gridmap glc_grid="mpas.gis20km" ocn_grid="EC30to60E2r2">
```

Comment:
Question for @jonbob  - Do the ICE mapping files below need to live in a separate `gridmap` entry that specifies `ice_grid` instead of `ocn_grid`?  Or is there a special handling that always assumes ICE grids match OCN grids, so that's not necessary?

Reply:
It is always assumed/required that ocn and ice are on the same grid